source/usage-examples/findOne.txt

```diff
@@ -3,3 +3,61 @@ Find a Document
 ===============
 
 .. default-domain:: mongodb
+
+You can retrieve a single document from your collection with the
+``collection.findOne()`` method. 
+
+The ``findOne()`` method takes a **context**, a **filter**, and a
+pointer to a configuration object as arguments.
+
+A context is a value of type ``context.Context``. Contexts are useful in
+concurrent server programming. A discussion of the context pattern and the
+``context`` package is outside the scope of this page, but if you are interested
+you can find more information
+`in this article on The Go Blog <https://blog.golang.org/context>`__.
+
+Your ``filter`` argument is your query to MongoDB. The ``filter`` parameter takes a query
+document that specifies the subset of your collection that you would like to
+retrieve. If you provide an empty document as your ``filter`` argument, MongoDB
+returns all the documents in your collection. The ``filter`` parameter has an empty
+interface type, but the value you pass as your ``filter`` argument must specify
+a document. For more information on documents in the MongoDB Go driver, see our
+:doc:`guide on data formats </fundamentals/data-formats>`.
+
+Your configuration object is used to specify any additional configuration
+options for your ``findOne()`` method call. A full list of supported options
+can be found in 
+:go-api:`the API documentation for the FindOneOptions type </mongo/options#FindOneOptions>`. 
```

Comment:
Thanks for the note. I thought in this case it would be helpful to readers as the `FindOneOptions` API docs are the best way to see how it can be used (complete list of options).

I removed the link and I agree it makes the page more readable. It is now only at the end.